### ISSUE TITLE
which axis to concatenate when setting is channel first and channel last

### ISSUE THREAD

**Issue by kaiche12** [fchollet/keras-docs-ja#129]
The Keras document on`Concatenate` said `concatenate(axis = -1)`. Does it work the same for both condition when `channel = first` and `channel = last` ?

For example I have three dataset with these shape `[None,1,16,8,8]`, `[None,1,16,8,8]` and `[None,1,16,8,8](batch_size,timestep,channel,height,width)`
Above setting channel is first

The same dataset with different channel position `[None,1,8,8,16]`, `[None,1,8,8,16]` and `[None,1,8,8,16](batch_size,timestep,height,width,channel)`
this setting channel is last

Now Keras document said concatenate more than two tensors instruct this way
`keras.layers.concatenate(axis=-1)`

**Comment by nzw0301** [comment]
First of all, sorry for being too late to reply.

This repo is for the Japanese translated site of Keras's document site, so, unfortunately, we cannot answer your question here.

Best regards,
